mix.exs

```diff
@@ -48,10 +50,14 @@ defmodule CheckerCab.MixProject do
       {:ecto, "~> 3.11"},
       {:ex_doc, "~> 0.31.2", only: :dev},
       {:dialyxir, "~> 1.4", only: [:dev, :test], runtime: false},
-      {:excoveralls, "~> 0.18", only: :test}
+      {:excoveralls, "~> 0.18", only: :test},
+      {:decimal, "~> 2.0"}
```

Comment:
I should have already called out that this is implicitly in our mix.lock file from Ecto.